app/submitter/converter.py

```diff
@@ -90,9 +91,12 @@ def convert_answers(schema, questionnaire_store, routing_path, flushed=False):
         payload['case_ref'] = metadata['case_ref']
 
     if schema.json['data_version'] == '0.0.3':
-        payload['data'] = convert_answers_to_payload_0_0_3(
-            answer_store, list_store, schema, routing_path
-        )
+        payload['data'] = {
+            'answers': convert_answers_to_payload_0_0_3(
+                answer_store, list_store, schema, routing_path
+            ),
+            'lists': convert_lists_to_payload_0_0_3(list_store),
```

Comment:
I don't think it's worth having this as a function - it could just as easily be:

```
'lists': list_store.serialise()
```